extensions/sql-migration/images/cutover.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="13" height="14" viewBox="0 0 13 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I don't see a screenshot for this change.

Reply:
![image](https://user-images.githubusercontent.com/6816294/122167212-7c5a2780-ce2f-11eb-9f8e-20bc5b152010.png)

Reply:
It replaces a play icon with this.